### PR TITLE
Correct table reference in repository documentation

### PR DESCRIPTION
**Pull request by madorb**
You refer to table 1 - which is the first table in THIS adoc file, but you only generate a single-page html file, so it, in fact is displaed as table 3.

Thank you for proposing a pull request. This template will guide you through the essential steps necessary for a pull request.
Make sure that:

- [ ] You have read the [Spring Data contribution guidelines](https://github.com/spring-projects/spring-data-build/blob/master/CONTRIBUTING.adoc).
- [ ] There is a ticket in the bug tracker for the project in our [JIRA](https://jira.spring.io/browse/DATACOUCH).
- [ ] You use the code formatters provided [here](https://github.com/spring-projects/spring-data-build/tree/master/etc/ide) and have them applied to your changes. Don’t submit any formatting related changes.
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] You added yourself as author in the headers of the classes you touched. Amend the date range in the Apache license header if needed. For new types, add the license header (copy from another file and set the current year only).
- [ ] You provide your full name and an email address registered with your GitHub account. If you’re a first-time submitter, make sure you have completed the [Contributor’s License Agreement form](https://support.springsource.com/spring_committer_signup).

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@madorb Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-data-couchbase&pullRequestId=129)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-data-couchbase&pullRequestId=129) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/faq) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@madorb Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-data-couchbase&pullRequestId=129)!

File: src/main/asciidoc/repository.adoc [hunk 1]
```diff
@@ -743,4 +743,4 @@ interface PersonRepository extends CrudRepository<Person, Long> {
 }
 ----
 
-Projections declare a contract between the underlying type and the method signatures related to the exposed properties. Hence it is required to name getter methods according to the property name of the underlying type. If the underlying property is named `firstName`, then the getter method must be named `getFirstName` otherwise Spring Data is not able to look up the source property.
\ No newline at end of file
+Projections declare a contract between the underlying type and the method signatures related to the exposed properties. Hence it is required to name getter methods according to the property name of the underlying type. If the underlying property is named `firstName`, then the getter method must be named `getFirstName` otherwise Spring Data is not able to look up the source property.
```
Comment (madorb) [inline src/main/asciidoc/repository.adoc]: Sorry for delay, updated the commit!

**Comment by simonbasle** [comment]
@subalakr I'll let you merge and cherry pick in the relevant branches (I think my view of branches is a tiny bit outdated 😜 )

**Comment by subalakr** [comment]
Merged

**Comment by subalakr** [comment]
Hi @madorb, we noticed your commit doesn't have your full name on it. Can you

**Comment by subalakr** [comment]
nvm, found it on your github account :)

**Pull request closed without merging**